module/zfs/vdev_label.c

```diff
@@ -1259,23 +1280,28 @@ vdev_label_sync(zio_t *zio, vdev_t *vd, int l, uint64_t txg, int flags)
 	 */
 	label = spa_config_generate(vd->vdev_spa, vd, txg, B_FALSE);
 
-	vp = zio_buf_alloc(sizeof (vdev_phys_t));
+	vp_abd = abd_alloc_for_io(sizeof (vdev_phys_t), B_TRUE);
```

Comment:
Since there's IO performed with this ABD I used `abd_alloc_for_io()`. I'm not really sure why this interface is introduced, and if metadata parameter will be used to allocate linear ABD...